content/industries.md

```diff
@@ -40,19 +40,3 @@ public interest. In the public sector, it is essential to maintain harmony and t
 entities. Additionally, timeliness of deployments and features may also be a strong consideration. The adoption of
 cloud native, along with modern methodologies, can increase organizational velocity, which is critical to many areas of
 the public sector.
-
-## Projects
-- [Project Calico](https://www.projectcalico.org/)
-- [SPIFFE/SPIRE](https://github.com/spiffe)
-- [OPA](https://github.com/open-policy-agent/opa)
-- [Istio](https://github.com/istio/istio)
-- [Linkerd](https://github.com/linkerd)
-- [HyperLedger](https://www.hyperledger.org/)
-- [HashiCorp Vault](https://github.com/hashicorp/vault)
-- [Keylime](https://keylime.dev/)
-- [Kyverno](https://github.com/kyverno/kyverno)
-- [Keycloak](https://github.com/keycloak/keycloak)
-
-## Examples
-- Adopt a Zero trust architecture with least-privilege security policies. See [this](zero-trust-architecture#examples)
-  for more examples.
```

Comment:
Just for my understanding, why did you remove this ?

Reply:
From the perspective of the reader, each industry has its challenges and these challenges needs to be solved with e.g. a CNCF project. But currently there is no link between an industry and a project, hence it was confusing finding a list of projects that is almost a one to one copy from zero-trust-architectures. So I believe there is not "just" zero trust, that is important for those industries but even more topics and these are all covered by other chapters.

Reply:
Thanks for the clarification.